server/src/client_service.rs

```diff
@@ -418,6 +455,47 @@ mod tests {
         assert_eq!(quotes2, quotes);
     }
 
+    #[test_with_logger]
+    fn submit_quotes_refuses_dust(logger: Logger) {
```

Comment:
Ideally there should be an SCI that does get accepted, to prove that the server is not always returning `UNSUPPORTED_SCI`. This does get validated in other tests so it's not a deal breaker, but I think it's nice for tests to show that the input have an effect on the test.